lib/main.dart

```diff
@@ -27,7 +46,7 @@ class Vritant extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      home: LandingPage(),
+      home: isFirstRun?OnBoardingPage():LandingPage(),
```

Comment:
Please format the code, so that it becomes more readable.
`isFirstRun ? OnBoardingPage() :  LandingPage()`

Reply:
Done!